recipes/libpcap/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "libpcap" %}
+{% set version = "1.8.1" %}
+{% set sha256 = "673dbc69fdc3f5a86fb5759ab19899039a8e5e6c631749e48dcd9c6f0c83541e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.tcpdump.org/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # For Windows, wait for libpcap-1.9 to be released. CMake support will be
```

Comment:
It would be preferable to open these things as issues in the feedstock once it is created...